addon/components/polaris-tag.js

```diff
@@ -56,19 +74,19 @@ export default Component.extend({
    * @type {String}
    * @default null
    */
-  buttonLabel: null,
-
-  handleMouseUpByBlurring,
+  buttonLabel: computed('tagText', function() {
+    return `Remove ${this.get('tagText')}`;
+  }).readOnly(),
 
-  updateButtonLabel() {
-    // Set the remove button's aria-label based on the current text in the tag.
+  updateTagText() {
+    // Read the tag text so we can use this for the aria-label.
     // We access the element's `textContent` so that this still works in block usage.
     let tagText = this.get('element.textContent') || '';
-    this.set('buttonLabel', `Remove ${tagText.trim()}`);
+    this.set('tagText', `${tagText.trim()}`);
```

Comment:
No need for the string template around `tagText.trim()` any more right?

Reply:
totally